plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/connection/ConnectionWizard.java

```diff
@@ -21,24 +21,32 @@
 
 	private final ConnectionWizardPage page;
 	
+	public static final String NEW_CONNECTION_TITLE = "New OpenShift Connection";
+	public static final String EDIT_CONNECTION_TITLE = "Edit OpenShift Connection";
+	
 	/**
 	 * Constructor to use when connecting with the default connection.
+	 * 
+	 * @param connection wizard shell title
 	 */
-	public ConnectionWizard() {
```

Comment:
I'd keep the default constructor that'd send ConnectionWizard.NEW_CONNECTION_TITLE as title, way less code changes that way